### DISCUSSION TITLE
Any insights on if this will work on a Dream Machine Pro SE?

### DISCUSSION THREAD

**Discussion by locmer** [fabianishere/udm-kernel-tools#28] [General]
I am looking to try this project but now see Ubiquity has a Dream Machine Pro SE in early access. I am unsing KPN fibre in the netherlands so I think I need to bits in the project to make it work. But is it likely to work with the SE version too? Thanks for the insights.

**Comment by fabianishere** [comment]
There is no support for the UDM Pro SE currently, since it uses different firmware from the old Dream Machine Pro.
You might be able to get it to work by manually recompiling the necessary files for the new firmware, but I can't guarantee that that will work. 

For IPTV, your best bet is probably to wait for Ubiquiti to release a new firmware with multicast routing support in the stock kernel. They already did so in the latest beta firmware for the old Dream Machine (Pro), so I expect that they will do the same for the UDM Pro SE in the future.

**Comment by redpoisonv3** [comment]
would it not be possible to port the se firmware to the pro?

**Reply by fabianishere** [reply]
It should be doable to port this project to the UDM SE, but I am unable to test it since I don’t own an SE myself.

Most of the work should be getting `kexec-mod` to compile against the UDM SE kernel sources, after that most of the work of this project can be re-used.

**Reply by redpoisonv3** [reply]
i just got a confirmation last night that we are getting a 2.x.x and a 3.x.x build based on bullseye, from UI-marcus
